scripts/azure-pipelines/linux-build_and_test.sh

```diff
@@ -18,16 +18,19 @@ function or_die () {
 }
 
 or_die cd axom
-git submodule init 
-git submodule update 
+git submodule init
+git submodule update
 
 echo HOST_CONFIG
 echo $HOST_CONFIG
 
+export BUILD_TYPE=${BUILD_TYPE:-Debug}
+
+
 if [[ "$DO_BUILD" == "yes" ]] ; then
     echo "~~~~~~ RUNNING CMAKE ~~~~~~~~"
-    or_die ./config-build.py -hc /home/axom/axom/host-configs/docker/${HOST_CONFIG}.cmake -DENABLE_GTEST_DEATH_TESTS=ON ${CMAKE_EXTRA_FLAGS}
-    or_die cd build-$HOST_CONFIG-debug
+    or_die ./config-build.py -hc /home/axom/axom/host-configs/docker/${HOST_CONFIG}.cmake -bt ${BUILD_TYPE} -DENABLE_GTEST_DEATH_TESTS=ON ${CMAKE_EXTRA_FLAGS}
+    or_die cd build-$HOST_CONFIG-${BUILD_TYPE,,}
```

Comment:
What do the two commas do?

Reply:
It converts the variable to lowercase -- https://askubuntu.com/a/383360

This was necessary because we're not passing in the build-path (`--bp`) in our azure pipelines
and `config-build.py` lower-cases the build type (`--bt`) here:
https://github.com/LLNL/axom/blob/3999cf27a5437b032f9d3e23ca4476c04d139a2e/config-build.py#L169-L177

Reply:
Thanks.